### DISCUSSION TITLE
[off topic] weird executables in a "sendto" folder that only works with drag&drop over them

### DISCUSSION THREAD

**Discussion by RubicBG** [ThioJoe/Windows-Super-God-Mode#32] [Q&A] [answered]
hey, I see you deal with "unusual things", but your methods for them are professional.
> .ZFSendToTarget
> .DeskLink
> .mydocs
> .MAPIMail

Is there a way to use these executables with cmd or ps commands? Тheir work is defined by CLSID in the registries, but using CLSID directly doesn't work. (at least I didn't manage to do anything)

**Comment by ThioJoe** [comment, marked as answer]
I actually have a video coming out next week where I mention those, but I'm not aware of any way to call them directly, it might be possible though. For example the .ZFSendToTarget calls `zipfldr.dll` and it might be possible to do it manually but I've read discussions online where people couldn't figure out how
